close out dusko grant

thelifeandtimes opened the pull request.

thelifeandtimes committed:
close out dusko grant

mikeosborne commented on content/grants/dusko-valve.md:
```diff
@@ -14,7 +14,7 @@ assignee = "~tasset-norlux"
 grant_id = "P0143"
 completed = false
-canceled = false
+canceled = true
```
```suggestion
canceled = false
```

mikeosborne commented on content/grants/dusko-valve.md:
```diff
@@ -14,7 +14,7 @@ assignee = "~tasset-norlux"
 champion = "~tacryt-socryp"
 grant_id = "P0143"
 completed = false
```
### 
```suggestion
completed = true
```

mikeosborne committed:
Modify metadata

Technically, the first milestone was complete/negotiated. We decided if any milestones are completed, the project would be marked as completed.

mikeosborne commented:
I thought we decided that a project with at least one completed milestone would be marked "complete = true"

thelifeandtimes committed:
Update content/grants/dusko-valve.md

Co-authored-by: mikeosborne <mike@urbit.org>

thelifeandtimes committed:
clean up and accept suggested changes

thelifeandtimes commented:
yup, that works. Just wasn't following how those play out here. should be gtg now.
